cmd/lambda/leases/get_test.go

```diff
@@ -1,66 +1,163 @@
 package main
 
 import (
-	"context"
-	"encoding/json"
-	"errors"
-	"net/http"
 	"testing"
 
-	"github.com/Optum/dce/pkg/api/response"
-	"github.com/Optum/dce/pkg/db"
-	"github.com/Optum/dce/pkg/db/mocks"
+	gErrors "errors"
+	"fmt"
+	"github.com/Optum/dce/pkg/config"
+	"github.com/Optum/dce/pkg/lease"
+	"github.com/Optum/dce/pkg/lease/leaseiface/mocks"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net/http/httptest"
+
+	"context"
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/stretchr/testify/require"
+	"net/http"
 )
 
 func TestGetLeaseByID(t *testing.T) {
 
-	t.Run("When the invoking Call and there are no errors", func(t *testing.T) {
-		expectdLease := &db.Lease{
-			ID:             "unique-id",
-			AccountID:      "123456789",
-			PrincipalID:    "test",
-			LeaseStatus:    db.Active,
-			LastModifiedOn: 1561149393,
+	type response struct {
+		StatusCode int
+		Body       string
+	}
+	tests := []struct {
+		name     string
+		expResp  response
+		leaseID  string
+		retLease *lease.Lease
+		retErr   error
+	}{
+		{
+			name:    "success",
```

Comment:
Making these more descriptive would make the test easier to parse. It's probably also good to be somewhat consistent with how descriptive the names of the non-table driven tests are below.

Reply:
Same as accounts.

Reply:
Modified test names.